warehouse/core/src/main/java/datawave/ingest/protobuf/Uid.java

```diff
@@ -1084,17 +895,15 @@ public Builder addQUARANTINEUIDBytes(com.google.protobuf.ByteString value) {
     
     private static com.google.protobuf.Descriptors.FileDescriptor descriptor;
     static {
-        java.lang.String[] descriptorData = {"\n\tUid.proto\022\030datawave.ingest.protobuf\"]\n"
+        java.lang.String[] descriptorData = {"\n\tUid.proto\022\030datawave.ingest.protobuf\"F\n"
                         + "\004List\022\016\n\006IGNORE\030\001 \002(\010\022\r\n\005COUNT\030\002 \002(\004\022\013\n\003"
-                        + "UID\030\003 \003(\t\022\022\n\nREMOVEDUID\030\004 \003(\t\022\025\n\rQUARANT"
-                        + "INEUID\030\005 \003(\tB\034\n\030datawave.ingest.protobuf" + "H\001"};
+                        + "UID\030\003 \003(\t\022\022\n\nREMOVEDUID\030\004 \003(\tB\034\n\030datawav" + "e.ingest.protobufH\001"};
         com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner assigner = new com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner() {
             public com.google.protobuf.ExtensionRegistry assignDescriptors(com.google.protobuf.Descriptors.FileDescriptor root) {
                 descriptor = root;
                 internal_static_datawave_ingest_protobuf_List_descriptor = getDescriptor().getMessageTypes().get(0);
                 internal_static_datawave_ingest_protobuf_List_fieldAccessorTable = new com.google.protobuf.GeneratedMessage.FieldAccessorTable(
-                                internal_static_datawave_ingest_protobuf_List_descriptor, new java.lang.String[] {"IGNORE", "COUNT", "UID", "REMOVEDUID",
-                                        "QUARANTINEUID",});
+                                internal_static_datawave_ingest_protobuf_List_descriptor, new java.lang.String[] {"IGNORE", "COUNT", "UID", "REMOVEDUID",});
                 return null;
             }
```

Comment:
What happens if the new Uid protobuf (with QUARANTINEUID no longer present) is used to deserialized a legacy serialized Uid protobuf (that had the QUARNTINEUID in its schema, but in unpopulated in practice)?

Reply:
I thought protocol buffers would just ignore unknown fields, but we should verify.

Reply:
@jschmidt10 verified that your understanding is correct.